cmds/core/gosh/gosh.go

```diff
@@ -168,6 +168,11 @@ func runInteractive(runner *interp.Runner, parser *syntax.Parser, stdout, stderr
 			break
 		}
 
+		if line == "nocomplete" {
```

Comment:
works for me though couldn't you set a variable that gets evaluated?

Reply:
if you can figure out how let me know, I don't find gosh all that easy to figure out.